.github/workflows/pkg.pr.new.yml

```
@@ -12,6 +12,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
```

Nope, the base branch didn't work, it did resolve to the right SHA-1, but the commit object is not in the tree, that means our checkout step isn't deep enough. Might need to tweak some options [here](https://github.com/actions/checkout).

---

Hold on, your initial suggestion of `HEAD HEAD^` might have been correct, due to the temporary merge commit GitHub creates to run CI workflows on:
https://www.kenmuse.com/blog/the-many-shas-of-a-github-pull-request/

---

If it doesn't work, and we want to explore the checkout step, we could try: 

```
- uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
  with:
    fetch-depth: 0  # This fetches all history for all branches and tags
```

Or: 

```
- uses: actions/checkout@v2
  with:
    fetch-depth: 0  # Fetches all history for all branches
    fetch-tags: true  # Fetches all tags
```

---

A fetch of 0 might be a bit overkill, if we fetch with a depth of 2, we can get the current HEAD (aka the temporary merge commit) and its first parent (aka the commit on the base branch where the PR would be merged).

So here:
```suggestion
      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
        with:
          fetch-depth: 2
```

And back to `HEAD HEAD^` in the git diff.